firerunner/src/main.rs

```diff
@@ -88,6 +121,8 @@ fn main() {
     let rootfs = cmd_arguments.value_of("rootfs").unwrap().to_string();
     let appfs = cmd_arguments.value_of("appfs");
     let cmd_line = cmd_arguments.value_of("command line").unwrap().to_string();
+    let mem_size = cmd_arguments.value_of("mem_size");
+    let vcpu_cnt = cmd_arguments.value_of("vcpu_cnt");
```

Comment:
`let vcpu_count`

we're not limited to 60 character line lengths and variable names are compiled away, so there no performance hit to longer names. The same could be said for `cmd_line`, `rootfs`, `mem_size`, etc, but `cnt` instead of `count` is maybe a bridge too far :)